plugins/source/aws/Makefile

```diff
@@ -1,7 +1,7 @@
 # Generate mocks for mock/unit testing 
-.PHONY: generate-mocks
-generate-mocks:
-	go generate ./client/services/...
+.PHONY: gen-mocks
+gen-mocks:
```

Comment:
Since we have other "gen-*" targets it only made sense to rename this to match those.